config/core/test-configs.yaml

```diff
@@ -448,6 +503,56 @@ test_plans:
       job_timeout: '30'
       priority: 0
 
+  ltp_riscv: &ltp_riscv
+    rootfs: debian_sid-ltp_nfs
+    pattern: 'ltp/{category}-{method}-{protocol}-{rootfs}-ltp-template.jinja2'
+    filters:
+      - blocklist: *kselftest_defconfig_filter
+    params: &ltp-params_riscv
+      skip_install: "true"
+      tst_cmdfiles: ""
+      job_timeout: '15'
+      skipfile: skipfile-lkft.yaml
+
+  ltp-crypto_riscv:
```

Comment:
I think you need to add `base_name: ltp-crypto` so the results are tracked as `ltp-crypto` and not a separate test plan just for RISC-V.  See `baseline_qemu` for example, the results are still plain `baseline`.

Reply:
Same thing for the other test plans btw.

Reply:
@gctucker I created a brand new PR for this (https://github.com/kernelci/kernelci-core/pull/1556) as this PR was more about choosing the debian version and not simply duplicating the test plans.

In the new PR, I fixed the base_name and added the test plans that will be initially run on the hifive platform